python/lsst/ts/standardscripts/auxtel/enable_atcs.py

```diff
@@ -102,20 +103,11 @@ def get_schema(cls):
                       - type: string
                       - type: "null"
                     default: null
+                ignore:
+                    description: CSCs from the group to ignore.
```

Comment:
Is this Hexapod:1 or hexapod_1. Please provide an example. I suggest using the former, because I think it is least surprising.